src/pybind/mgr/volumes/fs/operations/versions/metadata_manager.py

```diff
@@ -21,6 +21,7 @@
 
 class MetadataManager(object):
     GLOBAL_SECTION = "GLOBAL"
+    METADATA_SUBVOLUME_SECTION = "METADATA_SUBVOLUME"
```

Comment:
AIUI this section is exclusively for metadata explicitly set by users/admins on subvolumes? Maybe you can name them as
```
 SUBVOLUME_USER_METADATA_SECTION = "SUBVOLUME_USER_METADATA"
```
or something like that?

Reply:
+1

Reply:
It's also a good idea to use 'user metadata' in the place 'metadata' in the API documentation as well.

Reply:
Updated section name and documentation.